trajopt_optimizers/trajopt_optimizers_test_suite/src/basic_optimization_unit.cpp

```diff
@@ -0,0 +1,60 @@
+#include <trajopt_utils/macros.h>
+TRAJOPT_IGNORE_WARNINGS_PUSH
+#include <gtest/gtest.h>
+#include <iostream>
+
+#include <ifopt/problem.h>
+#include <ifopt/ipopt_solver.h>
+#include <console_bridge/console.h>
+TRAJOPT_IGNORE_WARNINGS_POP
+
+//Solver Files
+#include <trajopt_sqp/trust_region_sqp_solver.h>
+#include <trajopt_sqp/osqp_eigen_solver.h>
+
+//Problem Header Files
+#include <basic_optimization_unit.hpp>
+
+/**
+ * @brief Applies a joint position constraint and solves the problem with IPOPT
+ */
+TEST_F(BasicOptimization, basic_optimization_ipopt)
+{
+  ifopt::IpoptSolver solver;
+  // solver.SetOption("derivative_test", "first-order");
+  solver.SetOption("linear_solver", "mumps");
+   // ipopt.SetOption("jacobian_approximation", "finite-difference-values");
```

Comment:
Perhaps we should run the tests twice, once with finite-differences and once without.